src/js/components/NetworkVolumesComponent.jsx

```diff
@@ -0,0 +1,151 @@
+import classNames from "classnames";
+import React from "react/addons";
+
+import DuplicableRowControls from "../components/DuplicableRowControls";
+import DuplicableRowsMixin from "../mixins/DuplicableRowsMixin";
+import FormGroupComponent from "../components/FormGroupComponent";
+import TooltipComponent from "../components/TooltipComponent";
+
+import ExternalLinks from "../constants/ExternalLinks";
+
+var ExternalVolumesComponent = React.createClass({
```

Comment:
The file is called `NetworkVolumesComponent`. Let's stick to `ExternalVolumesComponent`.